allennlp/tests/fixtures/bidirectional_lm/characters_token_embedder.json

```diff
@@ -0,0 +1,79 @@
+{
+    "dataset_reader": {
+      "type": "conll2003",
+      "tag_label": "ner",
+      "token_indexers": {
+        "tokens": {
+          "type": "single_id",
+          "lowercase_tokens": true
+        },
+        "elmo": {
+          "type": "elmo_characters"
+        }
+      }
+    },
+    "train_data_path": "allennlp/tests/fixtures/data/conll2003.txt",
+    "validation_data_path": "allennlp/tests/fixtures/data/conll2003.txt",
+    "model": {
+      "type": "crf_tagger",
+      "text_field_embedder": {
+        "token_embedders": {
+          "tokens": {
+            "type": "embedding",
+            "embedding_dim": 50
+          },
+          "elmo": {
+            "type": "bidirectional_token_embedder",
+            "weight_file": "allennlp/tests/fixtures/bidirectional_lm/lm_weights.th",
+            "text_field_embedder": {
```

Comment:
I'm not terrifically thrilled with this config duplication. (Also used to train the bidirectional LM in the first place.) Is there a way around it?

Reply:
This helped a bunch. Thanks! It did require some restructuring, but it seems like a win.